New release of `openpyxl=3.1.0` brokes CI

Failed CI: https://github.com/modin-project/modin/actions/runs/4065167559/jobs/6999654783

Log:
```bash
...
modin\core\execution\dispatching\factories\factories.py:267: in _read_excel
    return cls.io_cls.read_excel(**kwargs)
modin\logging\logger_decorator.py:128: in run_and_log
    return obj(*args, **kwargs)
modin\core\io\file_dispatcher.py:154: in read
    query_compiler = cls._read(*args, **kwargs)
modin\logging\logger_decorator.py:128: in run_and_log
    return obj(*args, **kwargs)
modin\core\io\text\excel_dispatcher.py:141: in _read
    reader = WorksheetReader(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

>   ???
E   TypeError: __init__() takes exactly 6 positional arguments (5 given)

```

--- a/environment-dev.yml
+++ b/environment-dev.yml
@@ -19,7 +19,7 @@ dependencies:
   - s3fs>=2021.8
   - feather-format
   - lxml
-  - openpyxl
+  - openpyxl<3.1.0
   - xlrd
   - matplotlib
   - sqlalchemy>=1.4.0,<1.4.46

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -18,7 +18,7 @@ pytest-cov
 pytest-xdist
 feather-format
 lxml
-openpyxl
+openpyxl<3.1.0
 xlrd
 matplotlib
 sqlalchemy>=1.4.0,<1.4.46

--- a/requirements/env_hdk.yml
+++ b/requirements/env_hdk.yml
@@ -15,7 +15,7 @@ dependencies:
   - pyhdk==0.3.1
   - s3fs>=2021.8
   - psutil
-  - openpyxl
+  - openpyxl<3.1.0
   - xlrd
   - sqlalchemy>=1.4.0,<1.4.46
   - scipy

--- a/requirements/env_unidist.yml
+++ b/requirements/env_unidist.yml
@@ -14,7 +14,7 @@ dependencies:
   - s3fs>=2021.8
   - feather-format
   - lxml
-  - openpyxl
+  - openpyxl<3.1.0
   - xlrd
   - matplotlib
   - sqlalchemy>=1.4.0,<1.4.46

--- a/requirements/requirements-no-engine.yml
+++ b/requirements/requirements-no-engine.yml
@@ -12,7 +12,7 @@ dependencies:
   - s3fs>=2021.8
   - feather-format
   - lxml
-  - openpyxl
+  - openpyxl<3.1.0
   - xlrd
   - matplotlib
   - sqlalchemy>=1.4.0,<1.4.46

